### ISSUE TITLE
LED Brightness on 2477D dimmers?

### ISSUE THREAD

**Issue by biiiink** [nugget/python-insteonplm#148]
Is it possible to adjust the LED brightness on the 2477D dimmer switches?

**Comment by geoffdavis** [comment]
I don't believe so with this module, but it's probably not too hard to add the support.

Page 26 of the [Insteon Command Tables](http://cache.insteon.com/pdf/INSTEON_Command_Tables_20070925a.pdf) shows the command structure.

Per a [feature request I just raised](https://community.home-assistant.io/t/support-setting-insteon-keypadlinc-and-switchlinc-led-backlight-brightness-and-maybe-arbitrary-commands/161529) on the Home Assistant forums:

The Insteon Keypads and Switches/Dimmers support turning on and off the LED backlight ( standard commands 0x2008, 0x2009), as well as setting the LED backlight brightness (Extended command 0x2E00 0x0007 0x11-7F).

Indigo’s logs look like this when I send the command to set brightness to 1:

```
   Sent INSTEON                    "East Bedroom Hall Light" raw insteon command (20 09   00 00 00 00 00 00 00 00 00 00 00 00 00 00; ack: 09)
   Sent INSTEON                    "East Bedroom Hall Light" raw insteon command (2E 00   00 07 06 00 00 00 00 00 00 00 00 00 00 00; ack: 00)
```
And here are the logs when I send the command to set brightness to 75:

```
   Sent INSTEON                    "East Bedroom Hall Light" raw insteon command (20 09   00 00 00 00 00 00 00 00 00 00 00 00 00 00; ack: 09)
   Sent INSTEON                    "East Bedroom Hall Light" raw insteon command (2E 00   00 07 60 00 00 00 00 00 00 00 00 00 00 00; ack: 00)
```

That device, East Bedroom Hall Light, is a Switchlinc Dimmer (2-wire), but the commands to my KeypadLinc Dimmers look identical.

According to the Insteon command tables:

Page 8: The first command `0x2009` is a standard command commanding the device to turn on the back light. `0x2008` turns off the backlight completely, so it makes sense that you’d want it to be on in order for the backlight brightness to have any effect.

Page 26: The second command `0x2E00`, an extended command with _D1_ set to anything and _D2_ set to `0x07`, says set the backlight brightness to the value in the following data octet, _D3_. According to the doc, _D3_ must be in the range 0x11 through 0x7F
